package.json

```diff
@@ -58,7 +58,8 @@
       "import/prefer-default-export": "off",
       "import/no-unassigned-import": "off",
       "import/no-useless-path-segments": "off",
-      "no-console": "off"
+      "no-console": "off",
+      "no-func-assign": "off"
```

Comment:
This rule bugs me a bit. I prefer this:

```javascript
function foo() {
}
foo = wrap(foo)
```

Over this:

```javascript
const foo = wrap(function foo() { // don't want this to be anonymous because I want the name
})
```

Applies to all HOCs I use in React (like `React.memo`). In here we have this `wrapAsync` function which wraps all of our utilities in the config's `asyncWrapper` and I just like the way it looks better when we reassign the function.

Reply:
Why do you prefer the first one? Because I definitely didn't know you can re-assign function declarations. I wouldn't think this is valid which is exactly the rationale of this rule. The second pattern looks fine to me.

Reply:
In any case: If you like it why not disable it in kcd-scripts and explain the rationale? If you only have a personal, stylistic preference I'd always favor the default ruleset which makes onboarding easier.

Reply:
I've been meaning to disable it in eslint-config-kentcdodds for a while now 😅 Just haven't gotten around to it.

Reply:
I didn't know that was possible either